src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -1,6 +1,14 @@
 # This file defines the default (English) variants of all of the internationalised messages
 pscanalpha.desc = Passive Scan Rules - alpha
 
+pscanalpha.cookiesamesitescanner.name = Cookie Without SameSite Attribute
+pscanalpha.cookiesamesitescanner.desc=A cookie has been set without the SameSite attribute, which means that the cookie can be sent as a result of a 'cross-site' request. \
+The SameSite attribute is an effective counter measure to cross-site request forgery, cross-site script inclusion and timing attacks.
```

Comment:
I'm a fan of the oxford or serial comma. Not sure what you guys think, but I'd suggest one following "script inclusion"....just my 2cents.